src/Controls/src/Core/Registrar.cs

```diff
@@ -323,6 +323,28 @@ public static void RegisterStylesheets(InitializationFlags flags)
 			return properties;
 		}
 
+		internal static void RegisterEffects(Assembly[] assemblies)
```

Comment:
This moves the register effects code to its own method so it's not getting called along side the other registration calls